src/main/target/SITL/target.c

```diff
@@ -190,6 +209,47 @@ static void* udpThread(void* data) {
     return NULL;
 }
 
+static float readRCSITL(const rxRuntimeState_t *rxRuntimeState, uint8_t channel) {
+    UNUSED(rxRuntimeState);
+    return rcPkt.channels[channel];
+}
+
+static uint8_t rxRCFrameStatus(rxRuntimeState_t *rxRuntimeState)
+{
+    UNUSED(rxRuntimeState);
+    return RX_FRAME_COMPLETE;
+}
+
+
+static void* udpRCThread(void* data) {
+    UNUSED(data);
+    int n = 0;
+
+    while (workerRunning) {
+        n = udpRecv(&rcLink, &rcPkt, sizeof(rc_packet), 100);
+        if (n == sizeof(rc_packet)) {
+            if (!rc_received) {
+                printf("[data]new rc %d: t:%f AETR: %d %d %d %d AUX1-4: %d %d %d %d\n", n, rcPkt.timestamp,
+                    rcPkt.channels[0], rcPkt.channels[1],rcPkt.channels[2],rcPkt.channels[3],
+                    rcPkt.channels[4], rcPkt.channels[5],rcPkt.channels[6],rcPkt.channels[7]);
+                
+                rxRuntimeState.channelCount = SIMULATOR_MAX_RC_CHANNELS;
+                rxRuntimeState.rcReadRawFn = readRCSITL;
+                rxRuntimeState.rcFrameStatusFn = rxRCFrameStatus;
+
+                rxRuntimeState.rxRefreshRate = 20000;
+                rxRuntimeState.rxProvider = RX_PROVIDER_UDP;
+                rc_received = true;
+            }
+        }
+    }
+
+    printf("udpRCThread end!!\n");
+    return NULL;
+}
+
+
+
```

Comment:
UDP receiver, better for test SITL in WSL.

Reply:
Please remove extra lines

Reply:
@haslinghuis Removed.